json/Side_Text.txt

```diff
@@ -28412,270 +28412,270 @@
 		"jp_text": "ブルージーレクイエムさんは<%br>リーダーが好きなんですか？",
-		"tr_text": "",
+		"tr_text": "Bluesy Requiem, do you like Leader?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "あら、聞かなきゃ分からないほど<%br>適当なアプローチに見えたかしら。",
-		"tr_text": "",
+		"tr_text": "My, my. After all you've seen,<%br>did you really still have to ask?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "あ……ごめんなさい。<%br>そういう意味じゃなくて……",
-		"tr_text": "",
+		"tr_text": "Ah... sorry.<%br>In that case, are you saying...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "……分かってるわよぉ。<%br>からかってるだけに見えるんでしょ？",
-		"tr_text": "",
+		"tr_text": "...I understand where you're coming from.<%br>It must just look like harmless flirting to you.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "まあしょうがないわね。<%br>最初は確かにからかい半分だったしぃ。",
-		"tr_text": "",
+		"tr_text": "I'm no good at expressing how I really feel.<%br>Besides, it really was just a bit of fun at first.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "だけど今は本気よ。素敵な人じゃない。<%br>アタシだけのものにしたくなっちゃったわ。",
-		"tr_text": "",
+		"tr_text": "But now I'm serious, and I don't intend to play<%br>nice. Leader will be mine, and mine alone.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "それでね……メロッメロにして<%br>めちゃくちゃにしちゃうんだからぁ！",
-		"tr_text": "",
+		"tr_text": "Oh, yes... By the time I'm through, I'll have<%br>Leader wrapped around my little finger!",
 		"fileID": 28
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "な、な、なんだかよく分かんないですけど<%br>それはダメな気がします！",
-		"tr_text": "",
+		"tr_text": "I don't know why, but I have a<%br>really bad feeling about that!",
 		"fileID": 28
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "ジェネちゃん、分からないんだったら<%br>止めないでよぉ。",
-		"tr_text": "",
+		"tr_text": "Sweetie, don't try to interfere<%br>with things you don't understand.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "でも、リーダーに危険が<%br>迫っているような気がして……",
-		"tr_text": "",
+		"tr_text": "But it really feels like Leader's in danger...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "まあ、そうねえ。<%br>その言い分には一理あるのよね。",
-		"tr_text": "",
+		"tr_text": "Really, now?<%br>Where could you have got that idea?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "そうね……止めるって言うなら<%br>決着を付ける方法を考えておくわ。",
-		"tr_text": "",
+		"tr_text": "Well... If you're going to try and stop me,<%br>we'll have to work out how to settle this.",
```

Comment:
`But if you really want to try to stop me, well... We're just going to have to work this out.`